numpy/ma/tests/test_core.py

```diff
@@ -3789,19 +3789,19 @@ def test_toflex(self):
         assert_equal(record['_mask'], data._mask)
 
         ndtype = [('i', int), ('s', '|S3'), ('f', float)]
-        data = array([(i, s, f) for (i, s, f) in zip(np.arange(10),
+        data = array(list(zip(np.arange(10),
                                                      'ABCDEFGHIJKLM',
-                                                     np.random.rand(10))],
+                                                     np.random.rand(10))),
                      dtype=ndtype)
         data[[0, 1, 2, -1]] = masked
         record = data.toflex()
         assert_equal(record['_data'], data._data)
         assert_equal(record['_mask'], data._mask)
 
         ndtype = np.dtype("int, (2,3)float, float")
-        data = array([(i, f, ff) for (i, f, ff) in zip(np.arange(10),
+        data = array(list(zip(np.arange(10),
                                                        np.random.rand(10),
-                                                       np.random.rand(10))],
+                                                       np.random.rand(10))),
```

Comment:
This could use some cleanup.

Reply:
You mean indentation cleanup, don't you?